app-backend/app/src/main/scala/bucket/Routes.scala

```diff
@@ -1,7 +1,6 @@
 package com.azavea.rf.bucket
 
 import java.util.UUID
-import scala.concurrent.ExecutionContext
```

Comment:
This was leftover from the refactor, wasn't being used. I should probably remove it in a separate commit so it doesn't feel like a part of this effort.